orchagent/portsorch.cpp

```diff
@@ -789,6 +814,18 @@ bool PortsOrch::addPort(const set<int> &lane_set, uint32_t speed)
     attr.value.u32list.count = static_cast<uint32_t>(lanes.size());
     attrs.push_back(attr);
 
+    if(an == true) {
+        attr.id = SAI_PORT_ATTR_AUTO_NEG_MODE;
+        attr.value.booldata = true;
+        attrs.push_back(attr);
+    }
+
+    if(fec_mode != "") {
```

Comment:
can you make brace in a separate line and leave a space between if and parenthesis.